pkg/infoschema/infoschema_v2.go

```diff
@@ -263,9 +263,16 @@ func (is *infoschemaV2) TableByID(id int64) (val table.Table, ok bool) {
 		return nil, false
 	}
 
```

Comment:
How about add some comment here?

// If `itm` is found by search `is.byID`, it means there is no change on that table during schema version [itm.schemaVersion, is.schemaVersion], so it is safe to use that table of an earlier version.
// Search the cache for the table of that schema version.